app/models/article.rb

```diff
@@ -0,0 +1,2 @@
+class Article < ApplicationRecord
+end
```

Comment:
No validations? Are you happy with users creating articles without a `title` or a `content`?